modules/cimg/src/cimgutils.cpp

```diff
@@ -241,14 +244,14 @@ struct CImgNormalizedLayerDispatcher {
 struct CImgLoadLayerDispatcher {
     using type = void*;
     template <typename Result, typename DF>
-    void* operator()(void* dst, const std::string& filePath, uvec2& dimensions,
+    void* operator()(void* dst, std::string_view filePath, uvec2& dimensions,
                      DataFormatId& formatId, bool rescaleToDim) {
         using P = typename DF::primitive;
 
         const DataFormatBase* dataFormat = DF::get();
 
         try {
-            cimg_library::CImg<P> img(filePath.c_str());
+            cimg_library::CImg<P> img(SafeCStr{filePath}.c_str());
```

Comment:
```suggestion
            cimg_library::CImg<P> img(SafeCStr<255>{filePath}.c_str());
```
file paths can be longer than the default 120 chars.

Reply:
That is fine it will go to the heap then...